packages/core/data-transfer/src/strapi/providers/local-destination/index.ts

```diff
@@ -102,7 +102,8 @@ class LocalStrapiDestinationProvider implements IDestinationProvider {
   async close(): Promise<void> {
     const { autoDestroy } = this.options;
     this.transaction?.end();
-
+    assertValidStrapi(this.strapi);
+    this.strapi.db.lifecycles.enable();
```

Comment:
I haven't checked the code, but does this actually change something that could persist if this fails to run? Or is it just a nice thing to re-enable before shutting down the instance?

Reply:
it does have to run, or else strapi has to restart to have lifecycle hooks enabled

Reply:
or maybe not actually